src/common/graphHelpers.js

```diff
@@ -39,11 +39,12 @@ const getChartXAxisLabelIncrement = granularity => {
  * Return translated labels based on granularity.
  *
  * @param {string} granularity, see enum of rhelApiTypes.RHSM_API_QUERY_GRANULARITY_TYPES
+ * @param {string} tooltipLabel
  * @returns {Object}
  */
-const getGraphLabels = granularity => {
+const getGraphLabels = ({ granularity, tooltipLabel }) => {
   const labels = {
```

Comment:
a lot of this is just an attempt to genericize all these methods and make them reusuable for other data sets...

Reply:
After discussion we've decided that because of how close hypervisor updates are to being released we're going to roll with this set of changes... ignore my comments